student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
```

Comment:
- [ ] Ensure you import the person class for this inheritance to work

  ![person-err](https://user-images.githubusercontent.com/54848363/179598370-98ab538c-7b6f-4a19-a016-84034bd498c4.jpg)